src/utils/blockchain.test.js

```diff
@@ -0,0 +1,7 @@
+import { isEmptyProxy } from "./blockchain";
+
+test("isEmptyProxy works correctly", () => {
```

Comment:
What about null / undefined. If you check this code 
```javascript
promisify(objects.proxyRegistry.proxies)(account).then(r => isEmptyProxy(r) ? null : getProxyOwner(r).then(r2 => r2 === account ? r : null));
```

If we pass undefined or null it will continue as well so it will shoot for getProxyOwner with empty nullable value and it might cause some bugs.

Can you also write different scenarios like separate test  cases

` provide undefined value`
` provide null value`
`provide short zero address`
` provide full zero address`
`provide incomplete address`
`provide valid address`

Also yeah , you are not testing on of the cases. Where is the `0x00000....` case?